core/utils.py

```diff
@@ -94,6 +94,17 @@ class ExplorationConversionError(Exception):
     pass
 
 
+class ThrowGetError:
+    """Error class to throw error while invoking the get function."""
+
+    def __init__(self, message: str):
+        self.err_message = message
+
+    def get(self, *args: Any, **kwargs: Any) -> None:
+        """Get function throws error when invoked."""
```

Comment:
Just say what it does: "Raises an error when invoked."

Reply:
Done